resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,97 @@
+#                         Push Notification Implementation and Tracking Troubleshooting
+                         
+### Setting up Android/iOS app for receiving push notification includes 3 main steps:
+
+  {% tabs %} {% tab title="Android" %}
+
+
+   * **Configuring app in Google Firebase (Android)**: Go to the Google [Firebase Console](https://console.firebase.google.com/) and sign in using your Google Developer credentials. Add a project and add your app under that project using the application Id that you used in app. Read more [here](https://firebase.google.com/docs/android/setup#console) for setting up.
+
+  * Add generated **“google-service.json”** to project and set up the project and module level gradle files. 
+  [Refer](https://firebase.google.com/docs/android/setup#console) for setting up gradle files.
+  
+{% endtab %}
+
+{% tab title="iOS" %}
+
+  * Register app on Apple developer account and generate server certificate. [Refer](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/establishing_a_certificate-based_connection_to_apns?language=objc) for more details.
+{% endtab %} {% endtabs %}
+
+{% tabs %} { % tab title="Android" % }
+
+  * **Verify that Server key is configured in Campaign instance.**
+
+    Ensure that server key is successfully set up in Campaign instance.  
+    This can be verified in Campaign instance under following section Campaign -> Administrator -> Channels -> Mobile App (AEP SDK).  
+    Launch mobile app and check push channel settings, _it should say Android key set up success_. It should look like:
+    
+   ![alt text](https://github.com/shivam-tomar-sde/aep-sdks-documentation/blob/push-troubleshooting-document/.gitbook/assets/android_server_key.png "Server key configuration in Android.")
+
+{% endtab %}
+
+{% tabs %} { % tab title="iOS" % }
+
+  * **Verify that Server cerificate is configured in Campaign instance.**
+  
+      Ensure that server certificate (.pem file) is successfully set up in Campaign instance. This can be verified in Campaign instance under following section Campaign -> Administrator -> Channels -> Mobile App (AEP SDK). Launch mobile app and check push channel settings, it should say Android key set up success. It should look like:
```

Comment:
1. `Adobe Campaign -> Administration -> ...`
2. You are mentioning `it should say Android key set up success.` for iOS. Rephrase `Select the mobile app from the list and verify that the push channel settings status indicates iOS Certificate file is successfully uploaded.`